docs/guides/chip_tool_guide.md

```diff
@@ -1393,3 +1393,244 @@ $ ./chip-tool any read-by-id <cluster-ids> <attribute-ids> <destination-id> <end
+    Can be set to `null`.
+-   _<user-status\>_ can be set to `null` or to one of the following values:
+
+    -   `0` (`Available`) - This status indicates that the given user slot is
+        available for modification on the device. This value is selected is you
+        set `null` for the variable.
+    -   `1` (`OccupiedEnabled`) - This status indicates that the given user slot
+        is used and active.
+    -   `3` (`OccupiedDisabled`) - This status indicates that the given user
+        slot is used, but disabled. Unlike `0` and `1`, supporting this status
+        is optional.
+
+-   _<credential-rule\>_ is the number of credentials that must be used to
+    unlock the door lock. This parameter can be set to `null` or to one of the
+    following values:
+
+    -   `0` (Single) - One credential type is required to unlock. This value is
+        selected is you set `null` for the variable.
+    -   `1` (Dual) - Two credential types are required to unlock.
+    -   `2` (Triple) - Three credential types are required to unlock.
+
+-   _<destination-id\>_ is the ID of the door lock device.
+-   _<endpoint-id\>_ is the ID of the endpoint on the door lock device.
+-   `--timedInteractionTimeoutMs` is the timeout duration for the interaction,
+    in milliseconds (_<ms_value>_). This should allow enough time for the
+    interaction to complete.
+
+**Examples of command:**
+
+```
+$ ./chip-tool doorlock set-user 0 1 AAA 6452 1 0 0 1 1 --timedInteractionTimeoutMs 1000
+```
+
+```
+$ ./chip-tool doorlock set-user 0 1 null null null null null 1 1 --timedInteractionTimeoutMs 1000
+```
+
+### Step 2: Assign a credential
+
+Once you have a user created on the door lock device, use the following command
+pattern to assign a credential to it:
+
+```
+$ ./chip-tool doorlock set-credential <operation-type> <{Credential}> <credential-data> <user-index> <user-status> <user-type> <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
+```
+
+In this command:
+
+-   _<operation-type\>_ is one of the available types of operation for the
+    credential:
+
+    -   `Add` - This operation adds a new credential to a user at the slot at
+        _<user-index\>_.
+    -   `Clear` - This operation removes an existing credential from the user at
+        the slot at _<user-index\>_.
+    -   `Modify` - This operation modifies an existing credential for the user
+        at the slot at _<user-index\>_.
+
+-   _<{Credential}\>_ is a parameter in the JSON format, with the following two
+    fields:
+
+    -   _<credential-type\>_ is the type of the credential. It can have one of
+        the following values:
+
+        -   `0` - Programming PIN
+        -   `1` - PIN
+        -   `2` - RFID
+        -   `3` - Fingerprint
+        -   `4` - Finger vein
+
+    -   _<credential-index\>_ is the index of the credential, between `1` and
+        the maximum value you set in the `NumberOfCredentialsSupportedPerUser`
```

Comment:
"You" the person operating chip-tool do not set NumberOfCredentialsSupportedPerUser. The device does.